src/app.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const { router } = require('./routes/artist');
+const { albumRouter } = require('./routes/album');
+const app = express();
+
+app.use(express.json());
+
+app.use('/artists', router);
+app.use('/album', albumRouter);
```

Comment:
You have a few failing tests because of this line, it should be 'albums' not 'album':
```
app.use('/albums', albumRouter);
```